studio/app/optinist/wrappers/caiman/params/caiman_cnmf.yaml

```diff
@@ -1,11 +1,30 @@
-p: 1
```

Comment:
@quanpython 
With the addition of cnmfe, the existing cnmf parameters will also be updated.
Detailed information will be provided separately, but please add the parameters that need to be added to cnmf for this cnmfe support to `caiman_cnmf.yaml` as well.

Reply:
@quanpython 
This time, cnmf.yaml will basically remain as is.
Only the following parameters will be added.

- Aim (default: None)
- do_refit (default: False)
  - Related
    - https://github.com/arayabrain/barebone-studio/pull/59#discussion_r1315237608

Reply:
@itutu-tienday
Is  `Aim` a typo of `Ain`, or is it a new parameter?

Reply:
@quanpython 
Excuse me, yes, it is a typo in `Ain`.

Reply:
@quanpython 
> This time, cnmf.yaml will basically remain as is.
> Only the following parameters will be added.
> 
> - Ain (default: None)
> - do_refit (default: False)

What was the status of your response to the above?
The original caiman_cnmf.yaml has 11 parameters, but now it seems to have increased to about 20. Is there an intention to do so?

Reply:
@itutu-tienday 
Sorry, I misunderstood that the file needed to remain at that point.
I reverted it.